lib/CoreDisTools/.nuget/runtime.json

```diff
@@ -0,0 +1,19 @@
+{
+  "runtimes": {
+    "win7-x64": {
+      "Microsoft.DotNet.CoreDisTools": {
+        "toolchain.win7-x64.Microsoft.DotNet.Coredistools": "1.0.0-prerelease"
```

Comment:
Case should be aligned for all filenames/ids: "Coredistools" -> "CoreDisTools".
Please make sure this in the other cases.